fog/ledger/server/src/router_server.rs

```diff
@@ -49,6 +50,40 @@ where
     E: LedgerEnclaveProxy,
     RC: RaClient + Send + Sync + 'static,
 {
+    pub fn new_from_config(
```

Comment:
Can this be `#[cfg(test)]` or moved to a bare generator-style function in the unit test suite itself? It's effectively a way to create this object with the shards welded shut, which a footgun for anyone looking at this later (particularly w/o docs on why there's two constructors).